logic/src/commonMain/kotlin/com/wire/kalium/logic/feature/message/MessageSendFailureHandler.kt

```diff
@@ -54,4 +75,30 @@ class MessageSendFailureHandlerImpl internal constructor(
                         clientRepository.storeUserClientIdList(entry.key, entry.value)
                     }
             }
+
+    override suspend fun handleFailureUpdateMessageStatus(
+        failure: CoreFailure,
+        conversationId: ConversationId,
+        messageId: String,
+        messageType: String
+    ) {
+        when (failure) {
+            is NetworkFailure.FederatedBackendFailure -> {
+                kaliumLogger.i("Sending message of type $messageType failed due to federation context availability.")
```

Comment:
is `i` verbose enough? or should we upgrade it to `w` or even `e`rror level?

Reply:
There's another log right after that so it was just to inform about that particular type, but I changed it to `e`